### PR TITLE
Add OLMo April + July 2024 Checkpoints & Config

### PR DESCRIPTION
**Pull request by 2015aroras**

### CODE REVIEW

File: configs/official/OLMo-1.7-7B.yaml [hunk 1]
```diff
@@ -0,0 +1,1276 @@
...
+  num_workers: 32
+  drop_last: true
+  pin_memory: true
+  prefetch_factor: 8
+  persistent_workers: true
+  timeout: 0
+  paths:
+    ######### NON WEB DATA #########
+    # ~> GUTENBERG BOOKS (5.256 GT)
+    - s3://ai2-llm/preprocessed/olmo-mix/v1_6-decontaminated/books/gpt-neox-olmo-dolma-v1_5/part-0-00000.npy
```
Comment (2015aroras) [inline configs/official/OLMo-1.7-7B.yaml:222]: `https` data paths not yet available

**Comment by dumitrac** [comment]
@2015aroras - do you think that for 1.7 we should have two configs? I understand that training was stopped, config changed, then resumed.

**Comment by soheeyang** [comment]
Hi, thank you so much for the work! Is there an estimated date of when all the details including the https links of the training data and the data order files are going to be released?

**Comment by 2015aroras** [comment]
@kyleclo @soldni Do you have any updates on the data side?

File: configs/official/OLMo-1.7-7B.yaml (hunk 1 above)
Comment (2015aroras) [reply configs/official/OLMo-1.7-7B.yaml]: Done

**Pull request merged**